src/Hl7.Fhir.Core.Tests/Rest/FhirClientTests.cs

```diff
@@ -87,7 +87,7 @@ private static void CreateItems()
                         Family = "Donald",
                     }
                 },
-                Id = "pat1",
```

Comment:
Maybe a better approach to make the identifiers dependent on the FhirVersion, like for example:

```
"pat1" + ModelInfo.Version
```

Make this a variable and use them in the tests.